examples/tutorials/advanced/cartesian_histograms.py

```diff
@@ -0,0 +1,337 @@
+r"""
+Cartesian histograms
+====================
+
+Cartesian histograms can be generated using the :meth:`pygmt.Figure.histogram`
+method. In this tutorial, different histogram related aspects are addressed:
+
+- using vertical and horizontal bars
+- showing counts and frequency percent
+- adding annotations to the bars
+- showing cumulative values
+- using color and pattern as fill for the bars
+- using overlaid, stacked, and grouped bars
+"""
+
+# sphinx_gallery_thumbnail_number = 4
+
+
+# Import the required packages
+import numpy as np
+import pygmt
+
+###############################################################################
+# Generate random data from a normal distribution:
+
+np.random.seed(100)
+
+# Mean of distribution
+mean = 100
+# Standard deviation of distribution
+stddev = 20
+
+# Create two data sets
+data01 = mean + stddev * np.random.randn(42)
+data02 = mean + stddev * 2 * np.random.randn(42)
```

Comment:
Maybe we should use the [`np.random.normal`(https://numpy.org/doc/stable/reference/random/generated/numpy.random.normal.html) function?

Reply:
I am fine with this change, please see commit (https://github.com/GenericMappingTools/pygmt/pull/2445/commits/6378261c9bacb9b0f25407092e4a741fed7a3cb7). This way is probably a bit more elegant.

Reply:
Do we also want to update the Gallery example [Histogram](https://www.pygmt.org/dev/gallery/histograms/histogram.html)?

Hm, maybe we can add a link to this tutorial at the beginn of the [Gallery example - Histogram](https://www.pygmt.org/dev/gallery/histograms/histogram.htm)?